### PR TITLE
Add normalize argument to certain image metrics

### PR DESCRIPTION
**Pull request by SkafteNicki**
## What does this PR do?

Fixes https://github.com/Lightning-AI/metrics/issues/1244
Adds `normalize` argument to inception, kid, fid metric to allow passing in float images

## Before submitting

- [x] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/metrics/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to **update the docs**?
- [x] Did you write any new **necessary tests**?

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: src/torchmetrics/image/inception.py [hunk 1]
```diff
@@ -96,6 +98,7 @@ def __init__(
         self,
         feature: Union[str, int, Module] = "logits_unbiased",
         splits: int = 10,
+        normalize: bool = False,
```
Comment (Borda) [inline src/torchmetrics/image/inception.py:101]:
```suggestion
        normalized: bool = False,
```
`normalize` sounds to me as a normalize will be performed on the input in metric...

Comment (SkafteNicki) [reply src/torchmetrics/image/inception.py:101]: Just trying to keep it consistent with the other image metric which also has this kind of argument:
https://github.com/Lightning-AI/metrics/blob/169ec1e4e1773f4c993ef27356bbf06d4cab08da/src/torchmetrics/image/lpip.py#L101

Comment (Borda) [reply src/torchmetrics/image/inception.py:101]: I see, so in such a case would it make sense to have another PR with changing all args?

**Review by Borda — APPROVED** [review]

**Comment by Borda** [comment]
@justusschock ^^ pls :rabbit:
